server/admin/src/sidemenu/SideNav.js

```diff
@@ -49,7 +49,7 @@ const SideNav = ({ open, handleClose, handleOpen }) => {
     { label: 'Officials', link: '/officials' },
     { label: 'Publications', link: '/publications' },
     { label: 'Video Testimonials', link: '/videoTestimonials' },
-    { label: 'REST API Docs', link: 'http://localhost:5000/api-docs' },
```

Comment:
It looks like simply removing the host did not work. 

@markstur 
Do we need to use an environment variable so folks can set the location? And maybe not show it if it doesn't exist?

Reply:
@avoajaugochukwu , Thank you for the submission but it doesn't seem to work for me. If you have time would you try to add a `.env` file in the `{project-dir}/server/admin/` directory.  There should already be an `env.example` file there. You won't need to check in the `.env` file but we should add `REACT_APP_API_DOCS_LINK=http://localhost:5000/api-docs` to the `env.example` file to help others understand what is expected.

Then the SideNav.js would have something like:
`
const navItems = [
    { label: 'Legislative Artifacts', link: '/artifacts' },
    { label: 'Advocacy Groups', link: '/advocacyGroups' },
    { label: 'Categories', link: '/categories' },
    { label: 'Levels', link: '/levels' },
    { label: 'Geographies', link: '/geospatialDefinitions' },
    { label: 'Officials', link: '/officials' },
    { label: 'Publications', link: '/publications' },
    { label: 'Video Testimonials', link: '/videoTestimonials' },
  ]
  if (process.env.REACT_APP_API_DOCS_LINK && process.env.REACT_APP_API_DOCS_LINK !== '') {
    navItems.push({ label: 'REST API Docs', link: process.env.REACT_APP_API_DOCS_LINK })
  }
`

Please give that a try and if it works then push your code and this PR should auto update.

Thanks again!
-Ed